src/easydmp/eestore/migrations/0002_convert_to_jsonb.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.12 on 2020-10-16 10:27
+
+import django.contrib.postgres.fields.jsonb
+import django.core.serializers.json
+from django.db import migrations
+
+import json
+
+
+def convert_to_jsonblob(apps, schema_editor):
+    EEStoreCache = apps.get_model('eestore', 'EEStoreCache')
+    for ec in EEStoreCache.objects.all():
+        if isinstance(ec.data, str):
```

Comment:
And if not? Sorry if it's obvious.

Reply:
There *should* only be two possibilties: dict or dict serialized to str (but since you can write anything via the admin it could be "anything" in str...). Forward, we go from str to dict and need to do that explicitly (backwards it's handled for us). If it's already a dict, no need to do anything.

You might say that the "if" is redundant, but this way if there is garbage, if it can't be interpreted as a dict in a string it won't halt the conversion.

Reply:
More seriously, there is almost identical code in all three datamigrations. This code will be deleted again in step two so I have no intentions of making a one-use generalized function for it.